lib/utils/ConfigParser.js

```diff
@@ -289,13 +289,10 @@ class ConfigParser {
         }
 
         for (let pattern of patterns) {
+            const fileExtensions = ['.js', '.ts', '.feature', '.coffee', '.es6']
```

Comment:
Let's move this up to the top of the file, it should not be defined within the loop

Reply:
@christian-bromann Good point. Done.